pages/job-category/index.js

```diff
@@ -1,6 +1,6 @@
 import Link from "next/link";
 import { styled } from "styled-components";
-import JobList from "../components/JobList";
+import JobList from "../../components/JobList";
 import { v4 as uuidv4 } from "uuid";
 import tailwindConfig from "@/tailwind.config";
 import postcssConfig from "@/postcss.config";
```

Comment:
```suggestion
```
Not sure you're using any of these.

Reply:
I'm not using the last two..I´m unsure if I am using `import { v4 as uuidv4 ) from uuid ; `

Reply:
You'd be using it if you were using the `.map` array method somewhere in your code, which I don't think you are.